### ISSUE TITLE
Massive note missed for building from sources

### ISSUE THREAD

**Issue by runfaj** [soomla/unity3d-store#516]
It should be put all over the place in the instructions that building from sources requires android-19 and and android build tools 22.0.1. It should also be mentioned that ANDROID_HOME, ANT_HOME, and JAVA_HOME need to be defined in your paths configuration.

**Comment by dabeku** [comment]
Definitely. Without this issue I wouldn't have found out the reason why my source build didn't work.

**Comment by EugeneButusov** [comment]
Hi @runfaj @dabeku,
You're right, it should be clarified. I created ticket at SOOMLA Public Roadmap: https://trello.com/c/b8C1HMgh/205-docs-add-build-requirements-for-android-libraries
You can track issue status there and SOOMLA always very appreciate for any contribution :-)

**Comment by akessner** [comment]
I added ANDROID_HOME, ANT_HOME, and JAVA_HOME, and I added them to my path, but I still can't get the latest version to compile.

I'm using Java 1.8, and ElCapitan. Android and Ant installed through Brew.  After getting the following error, it can't seem to find the source files:

aidl:
     [echo] Compiling .aidl into java files...

compile.module.androidstoregoogleplay.production:
    [mkdir] Created dir: /Users/avik/Documents/projects/github/unity3d-store/submodules/android-store-google-play/out/production/AndroidStoreGooglePlay
    [javac] /Users/avik/Documents/projects/github/unity3d-store/submodules/android-store-google-play/module_androidstoregoogleplay.xml:110: warning: 'includeantruntime' was not set, defaulting to build.sysclasspath=last; set to false for repeatable builds
    [javac] Compiling 5 source files to /Users/avik/Documents/projects/github/unity3d-store/submodules/android-store-google-play/out/production/AndroidStoreGooglePlay
    [javac] warning: [options] bootstrap class path not set in conjunction with -source 1.6
    [javac] /Users/avik/Documents/projects/github/unity3d-store/submodules/android-store-google-play/src/com/soomla/store/billing/google/GoogleIabHelper.java:19: error: package android.app does not exist
    [javac] import android.app.Activity;
